CMakeLists.txt

```diff
@@ -25,3 +27,11 @@ option(EVM2WASM_TOOLS ON)
 if(EVM2WASM_TOOLS)
     add_subdirectory(tools)
 endif()
+
+install(DIRECTORY include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+
+install(
+    EXPORT evm2wasmTargets
```

Comment:
Is this a cmake level name? Can be used by externalproject to refer to stuff?

Reply:
This describes a "CMake package" but unfortunately this cannot be used in ExternalProject. This actually what Hunter workarounds.